lib/grpc/GrpcServer.ts

```diff
@@ -104,11 +104,11 @@ class GrpcServer {
    * @returns the cerificate and its private key
    */
   private generateCertificate = async (tlsCertPath: string, tlsKeyPath: string): Promise<{ tlsCert: string, tlsKey: string }> => {
-    const keys = pki.rsa.generateKeyPair(1024);
+    const keys = pki.rsa.generateKeyPair(2048);
     const cert = pki.createCertificate();
 
     cert.publicKey = keys.publicKey;
-    cert.serialNumber = String(Math.floor(Math.random() * 1024) + 1);
+    cert.serialNumber = String(Math.floor(Math.random() * 2048) + 1);
```

Comment:
I don't think this actually needs to change to use 2048 bit keys. Maybe we can make the serial number indicate that the cert is 2048 bit? Then we'd have an easy to check if existing certs are 2048 or 1024.

So something like this

```javascript
cert.serialNumber = `2048-${String(Math.floor(Math.random() * 100000000))}`;
```

That would give us serial numbers that look like "2048-" followed by a random number up to 8 digits. And I could check if a cert is 2048 bit just by seeing if the serial number starts with "2048".